src/libmongoc/src/mongoc/mongoc-stream-tls-secure-transport.c

```diff
@@ -37,6 +37,10 @@
 #include "mongoc-stream-private.h"
 #include "mongoc-stream-tls-secure-transport-private.h"
 #include <mcd-string.h>
+#include <common-macros-private.h>
+#include <mcd-atomic.h>
+
+BEGIN_IGNORE_DEPRECATIONS
```

Comment:
Unnecessary?

Reply:
I concur. Nothing in this file seems like it would trigger a deprecation warning.